conans/test/integration/conanfile/test_cpp_info_serialize.py

```diff
@@ -0,0 +1,40 @@
+import json
+import textwrap
+
+from conans.test.utils.tools import TestClient
+
+
+def test_cpp_info_serialize_round_trip():
+    """ test that serialize and deserialize CppInfo works
+    """
+    # TODO: Define standard name for file
+    c = TestClient()
+    conanfile = textwrap.dedent("""\
+        import os
+        from conan import ConanFile
+        from conan.tools import CppInfo
+
+        class Pkg(ConanFile):
+            name = "pkg"
+            version = "0.1"
+
+            def package(self):
+                cpp_info = CppInfo(self)
+                cpp_info.includedirs = ["myinc"]
+                cpp_info.libs = ["mylib", "myother"]
+                cpp_info.libdirs = ["mylibs"]
+                p = os.path.join(self.package_folder, "cpp_info.json")
+                cpp_info.save(p)
+
+            def package_info(self):
+                cpp_info = CppInfo(self).load("cpp_info.json")
```

Comment:
This test shows that the interface cannot be a class/static method like ``CppInfo.load()`` as we typically do in the codebase, but it is loading on an already existing object (overwriting it)